### PR TITLE
added import of ddf-ui documentation to docs module

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

imports Intrigue documentation content from ddf-ui repository. 

#### Choose 2 committers to review/merge the PR.

@jaymcnallie
@shaundmorris

#### How should this be ### tested?
content is included correctly in html and pdf versions

#### Screenshots (if appropriate)

[using.pdf](https://github.com/codice/alliance/files/7010067/using.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/pom.xml [hunk 1]
```diff
@@ -47,6 +47,9 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <properties>
+        <ddf-ui.version>5.0.8-SNAPSHOT</ddf-ui.version>
```
Comment (shaundmorris) [inline distribution/docs/pom.xml:51]: Should we plan on getting out a new release of DDF-UI?

**Review by shaundmorris — APPROVED** [review]

File: distribution/docs/pom.xml (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/pom.xml:51]: Potentially. it makes sense to me

**Review by jaymcnallie — APPROVED** [review]

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
closing, opened on wrong branch

**Pull request closed without merging**